### PR TITLE
Mobile version menu designe and make it functional with JavaScript

### PR DESCRIPTION
**Pull request by MuhammadUsmanAslam**
- Mobile menu designed with HTML CSS and made functional with JavaScript
- Updated Linters for JavaScript

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const showNav = document.getElementById('show-nav-btn');
+const hideNav = document.getElementById('hide-nav-btn');
+const mobileMenuLinks = document.getElementById('mobile-menu-links');
+const mobileMenu = document.getElementById('mobile-menu');
+
+showNav.addEventListener('click', () => {
+  if (mobileMenu.className === 'mobile-menu--hide') {
+    mobileMenu.className = 'mobile-menu--show';
+  }
```
Comment (GzimAsani) [inline main.js]: - [x] When the hamburger button is clicked and menu appears the background should be blur, kindly check the design _PS: You can make background blur similar to this on CSS **filter: blur(8px);**_
image
image

File: README.md [hunk 2]
```diff
@@ -158,19 +158,17 @@ You can deploy this project using: Github Pages
 - Twitter: [@twitterhandle](https://twitter.com/M_Usman_Aslam)
```
Comment (GzimAsani) [inline README.md:160]: -  [x] Under the USAGE section mention how can a user run your project IE clicking live on vs code

Comment (GzimAsani) [reply README.md:160]: - [x] Please either update or remove the FAQ section 
image

Comment (GzimAsani) [reply README.md:160]: - [x] Please fill these sections, your readme will look more professional 
image

Comment (GzimAsani) [reply README.md:160]: - [x] Remove this description please

File: style.css [hunk 3]
```diff
@@ -53,6 +59,39 @@ header {
...
+}
+
+.mobile-menu--hide {
+  display: none;
+  width: 0;
+  height: 0;
+}
+
+.mobile-menu--show {
+  display: flex;
```
Comment (GzimAsani) [inline style.css:78]: - [x] You have used `display: flex` 18 times which means you are repeating yourself a lot and that is bad CSS practice. I would suggest you create a class with display: flex in it and use it at the HTML components that you need 
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello @MuhammadUsmanAslam                             :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with creating the hamburger and it's functionality, it works great :+1: 

 ## Required Changes 
- [ ] Check the comments below :arrow_double_down: 


 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 4]
```diff
@@ -53,6 +63,40 @@ header {
...
+  font-family: 'Poppins', sans-serif;
+  font-weight: 600;
+  font-size: 2rem;
+  line-height: 44px;
+  backdrop-filter: blur(4px);
+}
+
+.mobile-menu--show > ul > li {
+  padding: 0.5em 0;
+}
```
Comment (binhussen) [inline style.css:97]: - Great job implementing previous code reviewer suggestions,👍 I think you missed the desktop version of your navigation, when updating other things , please fix it for better UI 👌



| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|||


😎️

🤝️

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @MuhammadUsmanAslam , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — APPROVED** [review]
Hi @MuhammadUsmanAslam,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**